src/Model.java

```diff
@@ -150,4 +154,7 @@ public boolean getIsLoadThrough(){
     return this.isLoadThrough;
   }
 
+  public Calculator getCalculator(){
+    return this.calculator;
```

Comment:
what does this do? why is model returning the calculator instead of the output?

Reply:
ok i will make it not return calculator